micrometer-core/src/test/java/io/micrometer/core/instrument/binder/mongodb/DefaultMongoCommandTagsProviderTest.java

```diff
@@ -97,4 +102,57 @@ private CommandSucceededEvent commandSucceededEvent(final int requestId) {
                 new BsonDocument(),
                 1200L);
     }
+
+    @Nested
```

Comment:
The choices were:

1. Just inline these tests and keep the longer name of "determineCollectionNameWith...`
2. Put these tests in a separate test class named something like `DefaultMongoTagsProviderDetermineCollectionNameTest`
3. Inline them but separate them clearly w/ a `@Nested` test class.

Wdyt?

Reply:
👍🏼